test/android/getDependencyConfig.spec.js

```diff
@@ -2,43 +2,49 @@ const path = require('path');
 const expect = require('chai').expect;
 const getDependencyConfig = require('../../src/config/android').dependencyConfig;
 const mockFs = require('mock-fs');
-const dependencies = require('../fixtures/dependencies');
+const mocks = require('../fixtures/android');
 
 describe('android::getDependencyConfig', () => {
 
-  before(() => {
-    mockFs({ testDir: dependencies });
-  });
+  before(() => mockFs({
+    empty: {},
+    nested: {
+      android: {
+        app: mocks.valid,
+      },
+    },
+    noPackage: {
+      android: {},
+    },
+  }));
 
   it('should return an object with android project configuration', () => {
     const userConfig = {};
-    const folder = path.join('testDir', 'valid');
+    const folder = 'nested';
 
     expect(getDependencyConfig(folder, userConfig)).to.be.an('object');
   });
 
   it('should return `null` if manifest file hasn\'t been found', () => {
     const userConfig = {};
-    const folder = path.join('testDir', 'empty');
+    const folder = 'empty';
 
     expect(getDependencyConfig(folder, userConfig)).to.be.null;
   });
 
   it('should return `null` if android project was not found', () => {
     const userConfig = {};
-    const folder = path.join('testDir', 'empty');
+    const folder = 'empty';
```

Comment:
in other places like this one we can omit creating a `folder` variable in a favor of using strings directly in the function invocation